api/v1beta1/ansibletest_types.go

```diff
@@ -29,6 +30,11 @@ type AnsibleTestSpec struct {
 	CommonOptions         `json:",inline"`
 	CommonOpenstackConfig `json:",inline"`
 
+	// The desired amount of resources that should be assigned to each test pod
+	// spawned using the AnsibleTest CR. https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements
+	// +kubebuilder:default:={limits: {cpu: "4000m", memory: "4Gi"}, requests: {cpu: "2000m", memory: "2Gi"}}
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
@eshulman2 , can you please 🙏 suggest the correct default Resource Limits for the ansible test pods? It will be possible to override the default value in the AnsibleTest CR.

Reply:
Response from @eshulman2 (through PM):

_... I would say it can be 2G RAM and 400M more less cpu and should work fine (depending on the amount of hosts and the concurrency)._